learn/getting-started/working-with-data/using-query-expressions-to-manipulate-data.md

```diff
@@ -0,0 +1,252 @@
+
+Let's call the `filterCountriesByCases` function from inside the `main` function.
+
+```ballerina
+public function main() {
+    string[] countries = filterCountriesByCases(covidTable, 10000000);
+    io:println("Countries with more than 10 million cases: ", countries);
+}
+```
+In the `main` function, you have called the `filterCountriesByCases` function and have provided `covidTable` and `10000000` as parameters so that the function will filter the countries, which have more than 10000000 covid cases. In the next line, you have printed the result from the function. Copy the above code onto `main.bal` and execute `bal run` from within the `query_expression` project folder.
+
+The result will print something similar to the below.
+
+```bash
+Compiling source
+        .../query_expressions:0.1.0
+
+Running executable
+
+Countries with more than 10 million cases: ["USA","India"]
+```
+## Finding the Top Three Countries By the Number Of Reported Covid Deaths
+
+Let's define a new function to find the top three countries with the highest number of Covid deaths. In this function, you will use queries to sort a table and retrieve a *limit*ed number of sorted records from the table.
+
+```ballerina
+public function findCountriesByHighestNoOfDeaths(table<CovidEntry> dataTable, int n) returns [string, decimal][] {
+    [string, decimal][] countriesWithDeaths = from CovidEntry entry in dataTable
+        order by entry.deaths descending
+        limit n
+        select [entry.country, entry.deaths];
+    return countriesWithDeaths;
+}
+```
+The `findCountriesByHighestNoOfDeaths` function uses queries to find the top three countries with the highest Covid deaths. The `order by` clause is used to sort the records in the table in `descending` order and limit the number of output records of the query to `n`. Here, as the result, the query produces an array of tuples of type `[string, decimal]`. A tuple contains the country and the number of reported deaths. The produced array is in descending order by the number of deaths.
+
+Let's call the `findCountriesByHighestNoOfDeaths` function from within the `main` function to find the top three countries by the number of deaths.
+
+```ballerina
+public function main() {
+    string[] countries = filterCountriesByCases(covidTable, 10000000);
+    io:println("Countries with more than 10 million cases: ", countries);
+
+    [string, decimal][] countriesWithDeaths = findCountriesByHighestNoOfDeaths(covidTable, 3);
+    io:println("Countries with highest deaths:", countriesWithDeaths);
+}
+```
+
+The last two lines were added to print the top three countries by the number of deaths. Copy the last two lines of the above code onto `main.bal` and execute `bal run` from within the `query_expression` project folder.
+
+The output will look like the below.
```

Comment:
```suggestion
The output will look like below.
```

Reply:
As per the point 20. I think we should keep `the below`